bin/o2a-run-sys-test

```diff
@@ -500,20 +511,58 @@ function submit_pig {
         --execute "${1}"
 }
 
-function convert {
+function convert_python {
     header "Convert workflow.xml to Airflow's ${OUTPUT_DAG_NAME}.py dag for example: ${LOCAL_APP_NAME}"
     if [[ ${WITH_COVERAGE} == "true" ]];  then
         coverage run -a \
-        "${LOCAL_O2A_BIN_SCRIPT}" -i "${LOCAL_APP_DIR}" -o "${LOCAL_OUTPUT_DIR}" -u "${DATAPROC_USER}" \
-           -d "${OUTPUT_DAG_NAME}"
+        "${LOCAL_O2A_BIN_SCRIPT}" \
+            --input-directory-path "${LOCAL_APP_DIR}" \
+            --output-directory-path "${LOCAL_OUTPUT_DIR}" \
+            --user "${DATAPROC_USER}" \
+            --dag-name "${OUTPUT_DAG_NAME}"
     else
-        "${LOCAL_O2A_BIN_SCRIPT}" -i "${LOCAL_APP_DIR}" -o "${LOCAL_OUTPUT_DIR}" -u "${DATAPROC_USER}" \
-           -d "${OUTPUT_DAG_NAME}"
+        "${LOCAL_O2A_BIN_SCRIPT}" \
+            --input-directory-path "${LOCAL_APP_DIR}" \
+            --output-directory-path "${LOCAL_OUTPUT_DIR}" \
+            --user "${DATAPROC_USER}" \
+            --dag-name "${OUTPUT_DAG_NAME}"
     fi
     python -m py_compile "${LOCAL_OUTPUT_DIR}/${OUTPUT_DAG_NAME}.py"
     footer
 }
 
+function convert_dot {
+    header "Convert workflow.xml to dot files for example: ${LOCAL_APP_NAME}"
+    "${LOCAL_O2A_BIN_SCRIPT}" \
+        --input-directory-path "${LOCAL_APP_DIR}" \
+        --output-directory-path "${LOCAL_OUTPUT_DIR}" \
+        --user "${DATAPROC_USER}" \
+        --dag-name "${OUTPUT_DAG_NAME}" \
+        --dot
+
+    set +x
+    if command -v dot > /dev/null && command -v imgcat > /dev/null; then
```

Comment:
I think that in case dot command exists, we should generate the .png file always. Then maybe we should try imgcat but it should be a separate step. There will be cases where people will have graphviz but they have no imgcat + we want the file to be stored and generated so that we can open it later.

Reply:
I updated the PR. Thanks.